src/Common/src/Internal/Padding.cs

```diff
@@ -1,22 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Runtime.InteropServices;
-
-namespace Internal
```

Comment:
The CoreLib version of this file has architecture specific ifdefs. The architecture specific ifdefs won't be respected in  CoreFX. 

This duplication was introduced intentionally in https://github.com/dotnet/coreclr/pull/17956 to avoid affecting performance until somebody can look into it more.